Source/Magick.NET/Shared/Profiles/Exif/ExifValue.cs

```diff
@@ -23,7 +23,7 @@ namespace ImageMagick
     public sealed class ExifValue : IEquatable<ExifValue>
     {
         private object _value;
-
+        
```

Comment:
Make sure you remove the whitespace at a later moment. StyleCop will complain about it.

Reply:
Next push.